zulipterminal/helper.py

```diff
@@ -666,3 +667,63 @@ def suppress_output() -> Iterator[None]:
     finally:
         os.dup2(out, 1)
         os.dup2(err, 2)
+
+
+def hash_util_encode(string: str) -> str:
```

Comment:
Let's use a different file for these, since we could combine these with the URL deconstruction too and/or put them into the API repo like that other method in future?